resources/views/components/menu/main.blade.php

```diff
@@ -83,7 +83,6 @@
             <x-dropdown-item :href="route('claims.active')">Sets in Progress</x-dropdown-item>
             <x-dropdown-item :href="route('game.random')">Random Set</x-dropdown-item>
             <x-dropdown-header>Hubs</x-dropdown-header>
-            <x-dropdown-item :href="url('gameList.php?s=6&c=100&f=1')">Hub List</x-dropdown-item>
             <x-dropdown-item :href="url('game/6914')">Central Hub</x-dropdown-item>
             <x-dropdown-item :href="url('game/9553')">Genre & Subgenre Hub</x-dropdown-item>
             <x-dropdown-item :href="url('game/5771')">Series Hub</x-dropdown-item>
```

Comment:
Should these links be updated to point directly to the hubs?

Reply:
I think probably not in this PR, but yes in the subsequent one.